lib/transport/multitransport.c

```diff
@@ -36,6 +36,10 @@ _do_transport_switch(MultiTransport *self, LogTransport *new_transport, const Tr
 {
```

Comment:
Add OneIdentity copyright to the file.
The correct copyright today is: `Copyright (c) 2024 One Identity LLC.`